src/cascadia/TerminalApp/App.cpp

```diff
@@ -1,899 +1,897 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
avoid line end change

Reply:
I think it is sorted now. Could you double-check in case I still did a mess up?